modules/report/includes/commerce_pos_report.pages.inc

```diff
@@ -884,14 +884,22 @@ function commerce_pos_report_build_journal_role_table(array $filters = array())
 
       $return_count = 0;
       $purchase_count = 0;
-
+	  $number_of_products = 0;
+	  
+	  $order_wrapper = entity_metadata_wrapper('commerce_order', $row->order_id);
```

Comment:
This should probably be moved into the switch statement below so it only gets triggered for purchases and not returns. Also the spacing needs some fixing.